models/category/src/category-draft/presets/sample-data-goodstore/serveware.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  LocalizedString,
+  KeyReference,
+} from '@commercetools-test-data/commons';
+import type { TCategoryDraftBuilder, TCategoryDraft } from '../../../types';
+import empty from '../empty';
+import kitchen from './kitchen';
+
+const kitchenDraft = kitchen().build<TCategoryDraft>();
+
+const serveware = (): TCategoryDraftBuilder =>
+  empty()
+    .name(
+      LocalizedString.presets
+        .empty()
+        ['en-GB']('Kitchen Collections')
+        ['en-US']('Kitchen Collections')
+        ['de-DE']('Alles zum Servieren')
+    )
+    .slug(
+      LocalizedString.presets
+        .empty()
+        ['en-GB']('kitchen-collections')
+        ['en-US']('kitchen-collections')
+        ['de-DE']('kitchen-collections')
```

Comment:
The slugs are different in the source data between DE and GB. Do we want them to be the same?

Reply:
fixed with [439a32c](https://github.com/commercetools/test-data/pull/357/commits/439a32c559fb3c5e19eec43ab4b2a07dc184ff10)